### PR TITLE
Update instructions for running tribler

### PR DESCRIPTION
**Pull request by mmetro**
The instructions for running tribler were missing some important steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,11 +45,18 @@ or, if you haven't added your ssh key to your github account:
 git clone --recursive  https://github.com/Tribler/tribler.git
 ```
 
+Second install the [dependencies](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Linux.md)
+
+Third run setup.py
+```bash
```
Comment (whirm) [inline README.md]: You don't need to install Tribler if you are running it from git, you can just execute the shell script as written in the next step.

File: README.md [hunk 2]
```diff
@@ -45,11 +45,18 @@ or, if you haven't added your ssh key to your github account:
 git clone --recursive  https://github.com/Tribler/tribler.git
 ```
 
+Second install the [dependencies](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Linux.md)
```
Comment (whirm) [inline README.md]: Could you make this link relative so it points to the right version of the document when in different branches?

**Comment by whirm** [comment]
Thanks for the PR and sorry for the delay, I'm currently on vacation :)

**Comment by mmetro** [comment]
I changed it to the relative link, thanks for getting back to me

**Comment by whirm** [comment]
Back from vacation! :D

Looks good, Could you squash the commits and I'll get it merged?

Thanks!

**Comment by whirm** [comment]
Nevermind, github has a squash and merge option now.

Thanks!

**Pull request merged**